app/routes/public/speakers.js

```diff
@@ -30,25 +30,69 @@ export const SPEAKERS_FILTER = [
 
 @classic
 export default class SpeakersRoute extends Route {
-  async model() {
+  queryParams = {
+    search: {
+      refreshModel: true
+    }
+  }
+
+  titleToken() {
+    return this.l10n.t('Speakers');
+  }
+
+  async model(params) {
     const eventDetails = this.modelFor('public');
+    const filterOptions = [
+      {
+        and: [
+          {
+            name : 'sessions',
+            op   : 'any',
+            val  : {
+              and: [
+                {
+                  name : 'deleted-at',
+                  op   : 'eq',
+                  val  : null
+                },
+                {
+                  or: [{
+                    name : 'state',
+                    op   : 'eq',
+                    val  : 'accepted'
+                  }, {
+                    name : 'state',
+                    op   : 'eq',
+                    val  : 'confirmed'
+                  }]
+                }
+              ]
+            }
+          }
+        ]
+      }
+    ];
```

Comment:
No need to redefine speaker filters, please delete

Reply:
but then speakers are filtered in index page also.

Reply:
Use shallow cloing

```js
const filterOptions = {...SPEAKER_FILTERS};
```